client/me/purchases/manage-purchase/purchase-meta.jsx

```diff
@@ -56,17 +56,22 @@ class PurchaseMeta extends Component {
 
 	renderPrice() {
 		const { purchase, translate } = this.props;
-		const { priceText, currencyCode, productSlug } = purchase;
+		// TODO: We need separate tax here, I think, for the price
+		const { priceText, currencyCode, productSlug, taxAmount } = purchase;
 		const period =
 			productSlug && isMonthly( productSlug ) ? translate( 'month' ) : translate( 'year' );
 
 		if ( isOneTimePurchase( purchase ) || isDomainTransfer( purchase ) ) {
-			return translate( '%(priceText)s %(currencyCode)s {{period}}(one-time){{/period}}', {
-				args: { priceText, currencyCode },
-				components: {
-					period: <span className="manage-purchase__time-period" />,
-				},
-			} );
+			return translate(
+				'%(priceText)s{{taxAmount}} %(currencyCode)s {{period}}(one-time){{/period}}',
```

Comment:
Hi! I've found a possible matching string that has already been translated **2** times:
`translate( '%(priceText)s %(currencyCode)s {{period}}/ %(period)s{{/period}}' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).